### PR TITLE
11 - hero block, review styles and dependency on default button decor…

### LINKED ISSUE #11: follow-up: hero block, review styles and dependency on default button decoration

### PR DESCRIPTION
**Pull request by chicharr**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #11

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/area-vip/partido-vip
- After: https://issue-11-hero-follow-up--realmadrid--hlxsites.hlx.page/area-vip/partido-vip

### CODE REVIEW

File: blocks/hero/hero.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+    const img = heroPic.querySelector('img');
+    heroPic.remove();
+    const optimizedHeroPic = createOptimizedPicture(img.src, img.alt, false, [{ media: '(min-width: 600px)', width: '2000' }, { width: '1200' }]);
+    block.append(optimizedHeroPic);
   }
   if (heroContent) {
     if (heroContent.parentElement) {
       heroContent.parentElement.remove();
+      // Remove button class from all except first link
+      Array.from(heroContent.querySelectorAll('.button'))
```
Comment (vdua) [inline blocks/hero/hero.js:16]: you can select the last two buttons using the following selector as well `heroContent.querySelectorAll('.button:not(:first-of-type)')`

File: blocks/hero/hero.css [hunk 2]
```diff
@@ -46,42 +46,33 @@ main .hero-content h1 {
...
 main .hero-content a:nth-of-type(2) {
-  background: unset;
-  margin: 0;
-  display: block;
   font-size: 18px;
   font-weight: 600;
   letter-spacing: -0.1px;
-  margin-top: 20px;
+  padding: 10px;
+  border: 2px solid transparent;
```
Comment (vdua) [inline blocks/hero/hero.css:71]: nitpick: border is not there in the original site.

**Comment by vdua** [comment]
There is some misalignment between the two sites still, though I am fine with it, since it is readable. I am not clear on the requirements from that perspective.

**Review by vdua — APPROVED** [review]

File: blocks/hero/hero.js (hunk 1 above)
Comment (chicharr) [reply blocks/hero/hero.js]: Done

**Pull request merged**